mod/profiles/set/right/report_search.rb

```diff
@@ -80,16 +137,32 @@ def research_group_report_query
     end
   end
 
+  def tab_listing content
+    card.variant = voo.structure if voo.structure
+    lazy_loading_tabs subvariant_tabs, selected_subvariant, content,
+                      type: "pills"
+  end
+
   [
     :claim,
     :metric,
     :project,
     :research_group,
     :source,
     :wikirate_company,
-    :wikirate_topic
+    :wikirate_topic,
+    :metric_value
   ].each do |cardtype|
     view "#{cardtype}_list" do
+      listing = render("#{cardtype}_sublist".to_sym)
+      return listing if card.research_group? || !subvariants
+      tab_listing listing
+    end
+
+    next if cardtype == :metric_value
+
+    view "#{cardtype}_sublist" do
+      card.variant = voo.structure if voo.structure
       default_listing
```

Comment:
I like this but might suggest breaking it into more methods (eg `define_tab_listing_where_applicable`)